readme.txt

```diff
@@ -23,6 +23,7 @@ This section describes how to install the plugin and get it working.
 == Changelog ==
 
 = Unreleased =
+* Add helper functions to detect the new Essential and Performance WooX plans. #1128
```

Comment:
To be super-future-proof, I think we should use `Woo Express` instead of `WooX`.
```suggestion
* Add helper functions to detect the new Essential and Performance Woo Express plans. #1128
```